Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -71,6 +71,16 @@
     shoes: ClothingShoesBootsCombatFilled
     gloves: ClothingHandsGlovesColorBlack
 
+# Syndicate Operative Outfit - NukeOps Reinforcement
+- type: startingGear
+  id: SyndicateOperativeGearReinforcementNukeOps
+  equipment:
+    jumpsuit: ClothingUniformJumpsuitOperative
```

Comment:
move to nukeops.yml, and probably give it a parent to reduce copypaste

Reply:
There kinda never was a nukeops.yml. I guess I could create one it might make sense. I don't know if that's in scope or not though.

Reply:
> There kinda never was a nukeops.yml. I guess I could create one it might make sense. I don't know if that's in scope or not though.

It's in Roles/Antags I believe

Reply:
I'll check it out

Reply:
so the Roles/Antags/nukeops.yml is for playtime requirements and not gear sets. This is a loadout and the other nukeops gear is in the same loadouts file.

Reply:
merge master, new version moves the starting gear to nukeops.yml

Reply:
oh nevermind I see what you mean, this one is a bit outdated

Reply:
Mission accomplished thanks for the help

Reply:
it's good but why is the nukeops starting gear in traitor.yml instead of nukeops

Reply:
I wanted it near the parent so it's easier to maintain (look one line up to see what I'm inheriting not search for a prototype somewhere else)

Reply:
k